lib/faye/reconnect/client_extension.rb

```diff
@@ -16,6 +16,7 @@ def initialize redis: nil, name:, on_handshake: nil
         @redis = EventMachine::Hiredis::Client.new(redis[:host], redis[:port], redis[:password], redis[:database])
         EM.schedule do
           @redis.connect
+          @redis.client('setname', name)
```

Comment:
should probably be "faye-reconnect/#{name}" or something like this, no? otherwise if the name is just "dispatcher" here we don't be able to distinguish the connections.

Reply:
This is the current output: 

`id=502 addr=127.0.0.1:40788 fd=8 name=dispatcher_domain-test age=19 idle=19 flags=N db=1 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=get
`

Reply:
exactly, that's not good, this is an additional connexion made by the faye-reconnecte extension just to store a cliend_id so we can reconnect later, it's not the main faye connection and we should be able to distinguish the two.